src/templates/_partials/_blog_post.twig

```diff
@@ -0,0 +1,16 @@
+<div class="pt-8 border-b border-gray-300 first:pt-0">
```

Comment:
Because this will be use in other contexts, it's best to rename this partial to something generic, like `_entry_post.twig`

Reply:
Also, make sure to check if variables are defined.